technology/web/src/main/java/com/github/mizool/technology/web/AbstractErrorHandlingFilter.java

```diff
@@ -18,29 +18,33 @@
 
 import java.io.IOException;
 
-import javax.inject.Inject;
 import javax.servlet.FilterChain;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-import com.github.mizool.core.rest.errorhandling.ErrorHandler;
+import lombok.extern.slf4j.Slf4j;
+
 import com.github.mizool.core.rest.errorhandling.ErrorMessageDto;
 import com.github.mizool.core.rest.errorhandling.ErrorResponse;
+import com.github.mizool.core.rest.errorhandling.ErrorResponseFactory;
 
+@Slf4j
 public abstract class AbstractErrorHandlingFilter extends HttpFilterAdapter
 {
-    @Inject
-    private ErrorHandler errorHandler;
+    private final ErrorResponseFactory errorResponseFactory;
+
+    protected AbstractErrorHandlingFilter()
+    {
+        this.errorResponseFactory = new ErrorResponseFactory();
+    }
 
     @Override
     public void doFilter(HttpServletRequest request, HttpServletResponse response, FilterChain chain)
         throws IOException
     {
         try
         {
-            TransactionalResponseWrapper transactionalResponseWrapper = new TransactionalResponseWrapper(response);
-            chain.doFilter(request, transactionalResponseWrapper);
-            transactionalResponseWrapper.commit();
+            chain.doFilter(request, response);
```

Comment:
Note for posterity: The transactional response wrapper is some of the oldest code we wrote. It came from "the usual" handling of responses in times of JSPs. Later, when we adapted streaming our responses, nobody thought of the fact that this actually completely breaks streaming. All responses are first built into the heap, then streamed from there.
Yes, this means that responses can abort mid-stream now, but you cannot have streaming and transactionality at the same time.